lib/ansible/module_utils/network/iosxr/utils/utils.py

```diff
@@ -11,6 +11,56 @@
 from ansible.module_utils.six import iteritems
 
 
+def remove_command_from_interface(interface, cmd, commands):
```

Comment:
The function doesn't seem specific to `interface`. Probably can be renamed to a more generic name something like `remove_command_from_list()`, `add_command_to_list()`

Reply:
renamed it as: `remove_command_from_config_list` and `add_command_to_config_list`